backend/spacewalk-backend.spec

```diff
@@ -1066,6 +1066,7 @@ rm -f %{rhnconf}/rhnSecret.py*
 %attr(755,root,root) %{_bindir}/mgr-sign-metadata
 %attr(755,root,root) %{_bindir}/mgr-sign-metadata-ctl
 %attr(755,root,root) %{_bindir}/spacewalk-diskcheck
+%attr(755,root,root) %{_bindir}/mgr-update-pkg-extra-tags.py
```

Comment:
I think you need to add this also to some Makefile to get it installed. Or is there a wildcard in place.

As this is a tool for one-time use, you could also think about to install it to "/usr/lib/susemanager/bin/" .
We have some more of them there.